modules/docs/arrow-docs/docs/docs/optics/dsl/README.md

```diff
@@ -73,14 +75,14 @@ import arrow.data.*
 val jane = Employee("Jane Doe", Company("Kategory", Address("Functional city", Street(42, "lambda street"))))
 val employees = Employees(listOf(john, jane).k())
 
-employees.setter().employees.every(ListK.each()).company.address.street.name.modify(String::capitalize)
+Employees.employees.every(ListK.each()).company.address.street.name.modify(employees, String::capitalize)
```

Comment:
Is it possible to provide `ListK.each()` automatically based on the property type internally so the user can just do?
```kotlin
Employees.employees.every().company.address.street.name.modify(employees, String::capitalize)
```
Also if we detect types for which we now a wrapper exists for example `List <-> ListK`, is it possible to preserve the user's type `List` while providing internally a `Each` instance for its wrapper type?

Reply:
@raulraja I tried to find an encoding for this but we need true typeclasses AFAIK.

We can do this only for one `F` due to type erasure and I am unsure if we should favor `List`/`ListK` by adding
`fun <T, A> Lens<T, ListK<A>>.every(): Traversal<T, A> = this compose ListK.traversal()`.

Because then we cannot do it for any other `F` like
`fun <T, A> Lens<T, SetK<A>>.every()` or `fun <T, K, A> Lens<T, MapK<K, A>>.every()`